crates/kitsune_p2p/mdns/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+name = "kitsune_mdns"
+version = "0.0.1"
+description = "p2p / mdns discovery framework"
+license = "Apache-2.0"
+homepage = "https://github.com/holochain/holochain"
+documentation = "https://github.com/holochain/holochain"
+authors = [ "Holochain Core Dev Team <devcore@holochain.org>" ]
+keywords = [ "holochain", "holo", "p2p", "mdns", "networking" ]
+categories = [ "network-programming" ]
+edition = "2018"
+
+[dependencies]
+log = "0.4"
+libmdns = "=0.6.0"
+mdns = "=3.0.0"
+futures-util = "0.3.1"
+futures-core = "0.3.1"
+async-stream = "0.2.0"
+base64 = "0.13"
+err-derive = "0.2.1"
+
+[dev-dependencies]
+env_logger =  { version = "0.8", default-features = false, features = ["termcolor","humantime","atty"] }
```

Comment:
We've switched to the [observability](https://crates.io/crates/observability) crate for console log output - https://github.com/holochain/holochain/blob/7c6a863454b0553b6aaacd4ec63bb9ad84d2264a/crates/holochain_websocket/Cargo.toml#L28